test/unit/generator/content/schedule/part3/index.test.js

```diff
@@ -0,0 +1,76 @@
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-table')
+const { getTable } = require('../../../../../../app/generator/content/schedule/part3/get-table')
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-reduction-summary')
+const getReductionSummary = require('../../../../../../app/generator/content/schedule/part3/get-reduction-summary')
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-top-up-summary')
+const getTopUpSummary = require('../../../../../../app/generator/content/schedule/part3/get-top-up-summary')
+
+const part3 = require('../../../../../../app/generator/content/schedule/part3')
+
+const { topUpSchedule, reductionSchedule } = require('../../../../../mocks/mock-schedule')
+
+describe('schedule part 3', () => {
```

Comment:
Can we use describes inside this for top-up and reduction tests to aid readability